App/Services/PhotoUtils.js

```diff
@@ -1,96 +1,78 @@
 import {
-  CameraRoll,
-  AsyncStorage
+  CameraRoll
 } from 'react-native'
 import RNFS from 'react-native-fs'
 import ImageResizer from 'react-native-image-resizer'
 import IPFS from '../../TextileIPFSNativeModule'
 
+export async function getPage (pageSize, cursor) {
+  let queryParams = { first: pageSize }
+  if (cursor !== null) {
+    queryParams['after'] = cursor
+  }
+  return CameraRoll.getPhotos(queryParams)
+}
+
+export async function getAllPhotos () {
+  let data = []
+  let hasNextPage = true
+  let cursor = null
+  let totalPages = 50 // I wanted to limit the total size of a return array
+  while (hasNextPage && totalPages) {
+    let photos = await getPage(25, cursor)
```

Comment:
figured it would save us from having to do big array compares by just limiting this to 50 pages of 25. could likely go a lot smaller, but we can test it out

Reply:
ended up reducing this to just 250 photos max per camera roll update.